node-graph/gcore/src/graphic_element/renderer.rs

```diff
@@ -423,10 +424,26 @@ impl GraphicElementRendered for VectorData {
 				Some(color) => peniko::Color::rgba(color.r() as f64, color.g() as f64, color.b() as f64, color.a() as f64),
 				None => peniko::Color::TRANSPARENT,
 			};
+			use crate::vector::style::{LineCap, LineJoin};
+			use vello::kurbo::{Cap, Join};
+			let cap = match stroke.line_cap {
+				LineCap::Butt => Cap::Butt,
+				LineCap::Round => Cap::Round,
+				LineCap::Square => Cap::Square,
+			};
+			let join = match stroke.line_join {
+				LineJoin::Miter => Join::Miter,
+				LineJoin::Bevel => Join::Bevel,
+				LineJoin::Round => Join::Round,
+			};
```

Comment:
The shape is getting cropped when a miter join is used. Be aware that the user can create arbitrarily sharp points with arbitrarily high Miter Limit values. The cropping thing will always need to be able to handle that.

Reply:
I did apply a gap of one line width, to the transform, If that is not sufficient we'll most likely have to add some sort of magic constant to account for the oversizing

Reply:
Definitely not a constant, since as I said, it can be arbitrarily sharp. It'll require some kind of equation to calculate.

Reply:
This is fixed and resolved